deep-learning/_posts/2017-02-27-unet.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: review
+title: U-Net Convolutional Networks for Biomedical Image Segmentation
+tags: deep-learning CNN segmentation medical essentials
+cite:
+  - Unet 
+  - https://arxiv.org/pdf/1505.04597.pdf
+---
+
+<style>
+  main.page-content p {
```

Comment:
Please replace `main.page-content` by `div.post-content`. It was a mistake to use `main.page-content` (it adds justification outside the text of the review)